### ISSUE TITLE
Make filter responsive

### ISSUE THREAD

**Issue by JoeyStk** [digitalfabrik/integreat-cms#1907]
### Motivation
In our attempt to make the CMS responsive I think I would make sense to also make the filters responsive. At the moment the filter input fields get squashed together on mobile screens.

### Proposed Solution
 I think for the filters it's enough to add breakpoints to the grid, and to make sure every input field has enough space to breathe.

### Alternatives
We could ask UI/UX about alternatives. 

### Additional Context
This is part of our Meta Issue to make the CMS responsive.

**Comment by seluianova** [comment]
@JoeyStk you mean, we could stack the filters vertically on top of each other, right?
Smthg like we're doing now with widgets in page form.
Do you think it can be a good first issue?
![image](https://github.com/digitalfabrik/integreat-cms/assets/115008338/71662ddc-c371-4257-b4ac-4fea6c7a82a5)

**Comment by JoeyStk** [comment]
Uff. To be honest I don't remember what I meant :sweat_smile: maybe we can ask @nikolahoff ?

**Comment by nikolahoff** [comment]
@hauf-toni did you work on this? I see it in figma as a backlog topic...

**Comment by hauf-toni** [comment]
@nikolahoff no, since it's a backlog topic, I haven't started working on it. you are welcome to take over the issue

**Comment by nikolahoff** [comment]
@JoeyStk we didn't work on that topic yet. So there are no designs yet. If it's an urgent topic we can prioritize it. Normally we talk about the prioritization in our monthly call with @osmers and @MizukiTemma. Let me know if the topic is urgent or if its ok to wait until Svenja is back from Holidays.

**Comment by JoeyStk** [comment]
Hi @nikolahoff. No worries, it can definitely wait until Svenja is back from vacation. It's not urgent :)

**Comment by JoeyStk** [comment]
Hi @nikolahoff, no worries at all. This is not urgent at all, we can wait until Svenja is back from vacation :)

**Comment by dkehne** [comment]
@osmers : not important. backlog

**Comment by osmers** [comment]
Stale
